testdata/src/main/java/com/hedvig/app/testdata/feature/insurance/builders/PerilBuilder.kt

```diff
@@ -0,0 +1,359 @@
+package com.hedvig.app.testdata.feature.insurance.builders
+
+import com.hedvig.android.owldroid.fragment.PerilFragment
+import com.hedvig.android.owldroid.graphql.InsuranceQuery
+import com.hedvig.android.owldroid.graphql.OfferQuery
+import util.toArrayList
+
+class PerilBuilder {
```

Comment:
There's a lot of unwarranted duplication in this file, I think we can manage to reduce it a bit.